ERCS/erc-7786.md

```diff
@@ -0,0 +1,184 @@
+---
+eip: 7786
+title: Cross-Chain Messaging Gateway
+description: An interface for contracts to send and receive cross-chain messages.
```

Comment:
Bit of a nitpick, but it might be nice to include what a "message" is in your description.